presto-parser/src/main/java/com/facebook/presto/sql/parser/AstBuilder.java

```diff
@@ -1934,6 +1937,11 @@ private static boolean isDistinct(SqlBaseParser.SetQuantifierContext setQuantifi
         return setQuantifier != null && setQuantifier.DISTINCT() != null;
     }
 
+    private static boolean ignoreNulls(SqlBaseParser.NullTreatmentContext nullTreatment)
```

Comment:
You can inline this. No need to have a private function since it's only used once.

Reply:
Done.